civihr_employee_portal/src/Forms/OnboardingWebForm.php

```diff
@@ -83,16 +80,34 @@ private function createReminderTask($contactID) {
    * If work email is included then create it here. It will be created on
    * webform submission anyway, but we need to make it primary.
```

Comment:
This description need to be updated to say what the function is now doing. It doesn't create an email anymore but updates the work email if it has previously been set by the webform submission.